client/src/pages/feedback.tsx

```diff
@@ -176,7 +176,11 @@ function FeedbackItem(props: {
             <Autocomplete
               key={`${feedback._id}-${feedback.updatedAt}`}
               data-cy="select-answer"
-              options={mentorAnswers || []}
+              options={
+                mentorAnswers?.filter(
+                  (mentorAnswer) => mentorAnswer.status == "COMPLETE"
```

Comment:
Instead of comparing to the literal string "COMPLETE", you should compare it to the enumerated value type for status, something like Status.COMPLETE

Reply:
Enumerators are data structures of constant length that hold a set of constant values